client/src/constants/imagine25/preSurveyQuestions.js

```diff
@@ -82,6 +82,12 @@ const preSurveyQuestions = [
     ],
     type: "multiChoice",
   },
+  {
+    question:
+      "Rank the following items in order of how important you view them in a software application. 1 being the most important.",
```

Comment:
Instead of "1 being the most important", what do you think about saying "(1 being the most important and 4 being the least important)" ?